romanisim/__init__.py

```diff
@@ -15,8 +15,8 @@
 import logging
 
 log = logging.getLogger(__name__)
-logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s',
-                    datefmt='%Y-%m-%d %H:%M:%S')
+# logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s',
+#                     datefmt='%Y-%m-%d %H:%M:%S')
```

Comment:
Should these be deleted?

Reply:
I like those but they are interacting with stcal's logging in a real way when --usecrds is on.  I should include some more fulsome documentation but let's leave that for the moment.  Thanks!